src/app_service/service/node/async_tasks.nim

```diff
@@ -0,0 +1,14 @@
+import status/status
```

Comment:
Sale already pointed it out, but we shouldn't rely on the old `status-lib` code. IMO, we can do it in another PR. Just need to open an issue to keep it in mind.